src/gui/accountsettings.cpp

```diff
@@ -687,20 +688,43 @@ void AccountSettings::slotScheduleCurrentFolderForceFullDiscovery()
 
 void AccountSettings::slotForceSyncCurrentFolder()
 {
+    auto isMetered = []() {
+        if (auto qNetInfo = QNetworkInformation::instance()) {
+            return qNetInfo->isMetered();
+        }
+        return false;
+    };
+
     if (auto selectedFolder = this->selectedFolder()) {
-        // Terminate and reschedule any running sync
-        for (auto *folder : FolderMan::instance()->folders()) {
-            if (folder->isSyncRunning()) {
-                folder->slotTerminateSync();
-                FolderMan::instance()->scheduler()->enqueueFolder(folder);
-            }
+        if (isMetered() && ConfigFile().pauseSyncWhenMetered()) {
```

Comment:
At the moment the menu item is disabled to I can't force a sync?

Reply:
fixed